.evergreen/config.yml

```diff
@@ -515,6 +515,18 @@ functions:
     - command: ec2.assume_role
       params:
         role_arn: ${aws_test_secrets_role}
+    - command: subprocess.exec
+      type: test
+      params:
+        binary: bash
+        include_expansions_in_env: [AWS_SECRET_ACCESS_KEY, AWS_ACCESS_KEY_ID, AWS_SESSION_TOKEN]
+        args: [src/.evergreen/csfle/setup.sh]
+    - command: subprocess.exec
+      type: test
+      params:
+        binary: bash
+        include_expansions_in_env: [AWS_SECRET_ACCESS_KEY, AWS_ACCESS_KEY_ID, AWS_SESSION_TOKEN]
+        args: [src/.evergreen/csfle/teardown.sh]
```

Comment:
Does this work? It looks like we are calling setup.sh then immediately calling teardown.sh. Shouldn't the teardown happen later after running the test suite?

Reply:
This is just to ensure that setup doesn't hang.  The test suite itself does its own setup and teardown for each python version.

Reply:
Can we add a comment explaining this and also the purpose of test-csfle.sh below?